src/views/Visualization.vue

```diff
@@ -1,11 +1,20 @@
 <template>
   <div id="visualization">
+    <h1>Delaware River Basin Temperature Prediction</h1>
     <!-- Eventually, stack all the components for the navigation bar in here -->
   </div>
 </template>
 
 <script>
-  
+  import Importance from "/1-Importance/Importance"
+
+  export default {
+    name: "Vizualization",
+    components: {
+      Importance: () => import( /* webpackPrefetch: true */ /*webpackChunkName: "nav"*/ "./components/Nav"),
+      Importance: () => import( /* webpackPrefetch: true */ /*webpackChunkName: "importance"*/ "./1-Importance/Importance")
```

Comment:
Should there be two `Importance` components?

Reply:
Oh right - this component (Visualization.vue) isn't working.  Currently everything's getting loaded into App.vue. So this currently doesn't effect anything, but was sloppy on my part regardless.